ldb/models.py

```diff
@@ -410,8 +410,8 @@ class Meta:
     first_year = models.IntegerField(_('first year'), blank=True, null=True)
     student_number = CharNullField(_('student number'), max_length=7, blank=True, null=True, unique=True)
 
-    emergency_name = models.CharField(_('emergency name'), max_length=48, blank=True)
-    emergency_phone = models.CharField(_('emergency phone'), max_length=16, blank=True)
+    emergency_name = models.CharField(_('emergency name'), max_length=48, blank=True, null=True)
```

Comment:
Waarom moeten deze nullable zijn (en dan geen `CharNullField`, maar dat terzijde)?

Reply:
Omdat we die data bij lange na niet van iedereen hebben.

Reply:
(En de data als import als `<null>` doorkomt en niet als '', wat in mijn ogen logischer is. Aangezien '' impliceert "bestaat niet" en `<null>` onbekend.

Reply:
Het is nu een `CharNullField`. Verder eens met mijn comment? We hebben veel van deze gegevens niet namelijk.